images/Dockerfile.medusa-backup-fromsrc

```diff
@@ -0,0 +1,18 @@
+FROM cassandra:3.11.5
+
+COPY medusa/cassandra-sudoer /etc/sudoers.d/
+
+COPY medusa/medusa-src /medusa-src
```

Comment:
This Dockerfile can probably be removed at some point. It's not included in the build.sh, but very useful to test new medusa versions from source.